packages/ui/src/components/templates/Dashboard/Chart/IndicatorChart.tsx

```diff
@@ -0,0 +1,196 @@
+
+  const commonStyles = useMemo(() => {
+    return getCommonStyles(Colors);
+  }, [Colors]);
+
+  const styles = useMemo(() => {
+    return getStyles(Colors);
+  }, [Colors]);
+
+  const containerWidth = useMemo(() => {
+    return getContainerWidth(widthGraph);
+  }, [widthGraph]);
+
+  const containerMinWidth = useMemo(() => {
+    return getContainerMinWidth();
+  }, []);
+
+  const renderIndicator = useMemo(() => {
+    if (datasets?.length === 1) {
+      return (
+        <View style={styles.containerOneValue}>
+          {!checkNullString(datasets[0].icon) && (
+            <Icon size={30} name={datasets[0].icon} />
+          )}
+          <View style={styles.containerValueTitle}>
+            <Text
+              style={styles.title}
+              textColor={Colors.primaryColor.background}
+              writingType="important">
+              {`${datasets[0].value} ${datasets[0].unit}`}
+            </Text>
+            <Text numberOfLines={2} style={styles.title}>
+              {datasets[0].title}
+            </Text>
+          </View>
+        </View>
+      );
+    } else if (datasets?.length > 1) {
+      return (
+        <>
+          <Text style={styles.groupTitle}>{title}</Text>
+          {datasets.map((data, index) => {
+            const isIconAbsent = data.icon == null;
+            return (
+              <View
+                style={[
+                  commonStyles.button,
+                  styles.valueContainer,
+                  data.color && {
+                    backgroundColor: data.color,
+                    borderColor: data.color,
+                  },
+                ]}
+                key={index}>
+                <View style={styles.groupIconValue}>
+                  {!isIconAbsent && (
+                    <Icon style={styles.icon} size={20} name={data.icon} />
+                  )}
+                  <Text numberOfLines={2} writingType="important">
+                    {data.title}
+                  </Text>
+                </View>
+                <View style={styles.textUnitContainer}>
+                  <TextUnit
+                    style={styles.textUnit}
+                    unit={data.unit}
+                    value={data.value}
+                    color={Colors.secondaryColor_dark}
+                    fontSize={16}
+                    numberOfLines={2}
+                    defaultColor={true}
+                  />
+                </View>
+              </View>
+            );
+          })}
+        </>
+      );
+    }
+  }, [Colors, commonStyles.button, datasets, styles, title]);
+
+  if (!Array.isArray(datasets) || datasets.length === 0) {
+    return null;
+  }
+
+  return (
+    <Container
+      style={[
+        styles.container,
+        {width: containerWidth, minWidth: containerMinWidth},
+        style,
+      ]}>
+      {renderIndicator}
+      {!checkNullString(title) && datasets?.length === 1 && (
+        <Text style={styles.title}>{title}</Text>
+      )}
```

Comment:
The title for datasets bigger than one is display in renderIndicator so it might be good to do the same to be consistent. Moreover, add `writingType="title"` (or important) to make it clear that it's a title